src/content/reference/react/components.md

```diff
@@ -22,3 +22,23 @@ React exposes a few built-in components that you can use in your JSX.
 ## Your own components {/*your-own-components*/}
 
 You can also [define your own components](/learn/your-first-component) as JavaScript functions.
+
+--
+
+## Client and Server Components {/*client-and-server-components*/}
```

Comment:
Can you make clear to people not using RSC what (if anything) to take away from this section?

Reply:
added a note. What do you think?